flocker/node/test/test_loop.py

```diff
@@ -434,6 +466,8 @@ def test_convergence_stop(self):
         # would make deployer.discover_local_state() to fail since it was
         # only configured with one result.
         action.result.callback(None)
+        reactor.advance(1.0)
```

Comment:
Would it make sense to assert that there are no pending calls on the clock (i.e. nothing will ever happen), instead of advancing and asking if anything has happened yet?